src/facade/dragonfly_connection.cc

```diff
@@ -431,6 +431,55 @@ string Connection::GetClientInfo(unsigned thread_id) const {
   return res;
 }
 
+string Connection::GetClientInfo() const {
+  auto res = GetClientInfo(0);
+  auto start = res.find("tid");
+  auto end = res.find("irq");
+  res.erase(start, end - start);
+
+  // The following are dummy fields and users should not rely on those unless
```

Comment:
Ok, apparently I am running an "old" version of the client (mine is `4.5`). 

I think this is fixed for `5.0`. We will need to update the version for pytests + test on `5.0`. Will ping about this on Monday.